tests/unit/class-wcsr-time-functions-test.php

```diff
@@ -223,6 +223,26 @@ public function provider_get_active_days_ratio() {
 				'billing_interval' => 1,
 				'expected_ratio'  => 0.99
 			),
+
+			// Jason's renewal test case - 12 day period, 13 days active, the ratio should account for 12 days because thats the days in period
+			21 => array(
+				'from_timestamp'   => '2017-09-14 14:21:40',
+				'days_in_period'   => 12,
+				'days_active'      => 13,
+				'billing_period'   => 'month',
+				'billing_interval' => 1,
+				'expected_ratio'  => 0.4
```

Comment:
nit pick, whitespace looks out in these arrays

Reply:
Fixed :) thanks